en/sql/function/regex_fn.rst

```diff
@@ -0,0 +1,727 @@
+    ==============================
+    0
+
+**Ranges** 
+
+To represent a range of characters, use the dash character (-) between two valid characters. 
+For example, "[a-z]" matches any alphabet letter whereas "[0-9]" matches any single number.
+
+.. code-block:: sql
+
+    SELECT ('adf' REGEXP '[a-f]');
+    SELECT ('adf' REGEXP '[g-z]');
+    
+::
+
+    ('adf' regexp '[a-f]')
+    ================================
+    1
+
+    ('adf' regexp '[g-z]')
+    ================================
+    0
+
+.. code-block:: sql
+
+    SELECT ('strike' REGEXP '^[^a-dXYZ]+$');
+    
+::
+
+    ('strike' regexp '^[^a-dXYZ]+$')
+    ================================
+    1
+
+**POSIX-based character classes**
+
+The POSIX-based character class (*[:classname:]*) defines categories of characters as shown below. [:d:], [:w:] and [:s:] are an extension to the ECMAScript grammar.
+
++------------+-----------------------------------------+
+| Class      | Description                             |
++============+=========================================+
+| [:alnum:]  | Alpha-numerical character               |
++------------+-----------------------------------------+
+| [:alpha:]  | Alphabetic character                    |
++------------+-----------------------------------------+
+| [:blank:]  | Blank character                         |
++------------+-----------------------------------------+
+| [:cntrl:]  | Control character                       |
++------------+-----------------------------------------+
+| [:digit:]  | Decimal digit character                 |
++------------+-----------------------------------------+
+| [:graph:]  | Character with graphical representation |
++------------+-----------------------------------------+
+| [:lower:]  | Lowercase letter                        |
++------------+-----------------------------------------+
+| [:print:]  | Printable character                     |
++------------+-----------------------------------------+
+| [:punct:]  | Punctuation mark character              |
++------------+-----------------------------------------+
+| [:space:]  | Whitespace character                    |
++------------+-----------------------------------------+
+| [:upper:]  | Uppercase letter                        |
++------------+-----------------------------------------+
+| [:xdigit:] | Hexadecimal digit character             |
++------------+-----------------------------------------+
+| [:d:]      | Decimal digit character                 |
++------------+-----------------------------------------+
+| [:w:]      | Word character                          |
++------------+-----------------------------------------+
+| [:s:]      | Whitespace character                    |
++------------+-----------------------------------------+
+
+**Character equivalents** 
+
+The character equivalents (*[=word=]*) indicates that contain characters should be considered as identical for sorting.
+
+.. note::
+
+  **Compatibility Considerations**
+
```

Comment:
How about writing at the top?
It would be nice if it was written in the form of 'NOTE' in the link below.
https://www.cubrid.org/manual/en/10.2/sql/tuning.html#updating-statistics

Reply:
good point, thank you. I moved it at the top of the section.